### ISSUE TITLE
Protein expression: info missing when transporter direction is not in DB

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/PK-Sim#2034]
1) Create new project
2) Add individual
3) Add transporter protein from Dababase: OATP1B1

In PK-Sim 9:

![image](https://user-images.githubusercontent.com/25903040/148784158-0e638bbb-b668-4d4b-a66a-e2ce319d4a05.png)

In PK-Sim 10, the hint is missing:

![image](https://user-images.githubusercontent.com/25903040/148784527-6373c94a-b18b-4787-b087-e8f3dc8e80b7.png)

**Comment by msevestre** [comment]
We used to set apical basolateral etc... automatically. We do not anymore. Therefore, I am not sure that this message makes sense.

**Comment by PavelBal** [comment]
This message makes very much sense as it is about the direction, not localization. If I add expression of OATP1B1 in V9, I see a warning that "Efflux" was set by default and this might not be correct. So I would know that this is my responsibility to check the direction (it actually must be changed to "Influx").

If I add the same transporter in V10, I do not see this warning and would think that "Efflux" is coming from the DB and is the correct direction.

I realized this as I was going through the released models and was wondering why OATP1B1 is set to "Influx" there instead of "Efflux" as given by the DB. Only as I tried to rebuild this in V9 I saw that "Efflux" is as default value and not coming from the DB.
